Kernel/Syscalls/process.cpp

```diff
@@ -64,4 +64,17 @@ int Process::sys$set_process_name(Userspace<const char*> user_name, size_t user_
     return 0;
 }
 
+// FIXME: If we can get a syscall function with 5 arguments, we could pass key/value here.
+int Process::sys$set_coredump_metadata(Userspace<const char*> data, size_t data_length)
+{
```

Comment:
I was thinking whether this needs a `REQUIRE_PROMISE()` but I figured not, otherwise all processes using pledge() would have to use it or their coredumps won't contain assertion messages...

Reply:
more like they'd crash because they crashed, so they'd crash because they crashed, ...